frontend/src/container/LogDetailedView/ActionItem.tsx

```diff
@@ -78,7 +80,7 @@ function ActionItem({
 				...(idStart ? { idGt: idStart } : {}),
 				...(idEnd ? { idLt: idEnd } : {}),
 			});
-			getLogsAggregate({
+			getLogsAggregateProp({
```

Comment:
`getLogsAggregate` will throw a no-shadow error, should I rename it to something like `fetchLogsAggregate` or `retrieveLogsAggregate` to avoid no-shadow and bypass the global variable?